app/helpers/application_helper.rb

```diff
@@ -5,10 +5,10 @@ def options_for_video_ratings(selection = nil)
 end
 
 def this_video_average_rating(some_video)
-  if some_video.average_rating != 0
-   return "#{some_video.average_rating}/5"
-   else
-   return "Unrated"
+  if some_video.average_rating == 0
+    return "Unrated"
```

Comment:
You don't need the `return` here. There is implicit return. I would also suggest supplying the average rating instead of the whole video. The helper doesn't need to be concerned with how the rating is derived from the video, all this should do is out put either `Unrated` or #{rating}/5. You could probably just do a ternary for that.

```ruby
def display_rating(rating)
  rating == 0 ? "Unrated" : "#{rating}/5"
end
```